tests/kuttl/tests/change_keystone_config/00-deploy_keystone.yaml

```diff
@@ -0,0 +1 @@
+../../common/keystoneapi_deploy.yaml
```

Comment:
should there always be cleanup be triggered as 00 to make sure we start with fresh sample deployment?

Reply:
Done